assets/admin.js

```diff
@@ -81,4 +81,20 @@ window.addEventListener("load", () => {
 			}
 		}
 	});
+
+	// Open accordions with errors on submit.
+	document.addEventListener("ea.form.error", (event) => {
+		const { page, form } = event.detail;
+		// alert(`The ${page} form contains errors. Please resolve these before submitting again.`)
```

Comment:
Saved for later?

Reply:
No, it's a leftover from testing example code in https://github.com/EasyCorp/EasyAdminBundle/pull/6650. I've removed the line.